tests/Microsoft.Windows.Setting.Accessibility/Microsoft.Windows.Setting.Accessibility.Tests.ps1

```diff
@@ -1,141 +1,161 @@
-        $testResult.InDesiredState | Should -Be $true
-
-        # Invoking set should not change these values.
-        Invoke-DscResource -Name MousePointer -ModuleName Microsoft.Windows.Setting.Accessibility -Method Set -Property $parameters
-        $finalState = Invoke-DscResource -Name MousePointer -ModuleName Microsoft.Windows.Setting.Accessibility -Method Get -Property @{}
-        $finalState.PointerSize | Should -Be $initialState.PointerSize
-    }
-
-    It 'Sets desired value' {
-        # Randomly generate desired state. Minimum is set to 1 to avoid KeepCurrentValue
-        $desiredPointerSize = [PointerSize](Get-Random -Maximum 4 -Minimum 1)
-
-        $desiredState = @{ PointerSize = $desiredPointerSize }
-      
-        Invoke-DscResource -Name MousePointer -ModuleName Microsoft.Windows.Setting.Accessibility -Method Set -Property $desiredState
-   
-        $finalState = Invoke-DscResource -Name MousePointer -ModuleName Microsoft.Windows.Setting.Accessibility -Method Get -Property @{}
-        $finalState.PointerSize | Should -Be $desiredPointerSize
-    }
-}
-
-Describe 'VisualEffect'{
-    It 'AlwaysShowScrollbars.'{
-        Invoke-DscResource -Name VisualEffect -ModuleName Microsoft.Windows.Setting.Accessibility -Method Set -Property @{ AlwaysShowScrollbars = $false }
-
-        $initialState = Invoke-DscResource -Name VisualEffect -ModuleName Microsoft.Windows.Setting.Accessibility -Method Get -Property @{}
-        $initialState.AlwaysShowScrollbars | Should -Be $false
-
-        # Set 'AlwaysShowScrollbars' to true.
-        $parameters = @{ AlwaysShowScrollbars = $true }
-        $testResult = Invoke-DscResource -Name VisualEffect -ModuleName Microsoft.Windows.Setting.Accessibility -Method Test -Property $parameters
-        $testResult.InDesiredState | Should -Be $false
-
-        # Verify the changes are correct.
-        Invoke-DscResource -Name VisualEffect -ModuleName Microsoft.Windows.Setting.Accessibility -Method Set -Property $parameters
-        $finalState = Invoke-DscResource -Name VisualEffect -ModuleName Microsoft.Windows.Setting.Accessibility -Method Get -Property @{}
-        $finalState.AlwaysShowScrollbars | Should -Be $true
-
-        $testResult2 = Invoke-DscResource -Name VisualEffect -ModuleName Microsoft.Windows.Setting.Accessibility -Method Test -Property $parameters
-        $testResult2.InDesiredState | Should -Be $true
-    }
-}
-
-AfterAll {
-    $env:TestRegistryPath = ""
-}
\ No newline at end of file
+
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+using module Microsoft.Windows.Setting.Accessibility
+
+$ErrorActionPreference = "Stop"
+Set-StrictMode -Version Latest
+
+<#
+.Synopsis
+   Pester tests related to the Microsoft.Windows.Setting.Accessibility PowerShell module.
+#>
+
+BeforeAll {
+    Install-Module -Name PSDesiredStateConfiguration -Force -SkipPublisherCheck

```

Comment:
It's _probably_ better to install-if-needed rather than force install on each run of the test suite.

```suggestion
    if ($null -eq (Get-Module -ListAvailable -Name PSDesiredStateConfiguration)) {
        Install-Module -Name PSDesiredStateConfiguration -Force -SkipPublisherCheck
    }
```

Reply:
cc @ryfu-msft

Reply:
Changed to suggested